embeddings/embedding/static/flair.py

```diff
@@ -0,0 +1,159 @@
+from typing import Optional
+
+import flair
+import gensim
+import numpy as np
+import torch
+
+from flair.embeddings import WordEmbeddings
+from flair.file_utils import cached_path
+from pathlib import Path
+from pickle import UnpicklingError
+from torch import nn
+
+
+class WordEmbeddingsPL(WordEmbeddings):
+    """Standard static word embeddings, such as GloVe or FastText for polish language."""
+
+    def __init__(
+        self,
+        embeddings: str,
+        field: str = None,
+        fine_tune: bool = False,
+        force_cpu: bool = True,
+        stable: bool = False,
+    ):
+        """
+        Initializes classic word embeddings. Constructor downloads required files if not there.
+        :param embeddings: one of: 'glove', 'extvec', 'crawl' or two-letter language code or custom
+        If you want to use a custom embedding file, just pass the path to the embeddings as embeddings variable.
+        set stable=True to use the stable embeddings as described in https://arxiv.org/abs/2110.02861
+        """
+        super(WordEmbeddings, self).__init__()
```

Comment:
I think in Python 3 ```super().__init__()``` is enough and preferred 👌

Reply:
is not enough, then we need to pass all params to init what leads to UnpicklingError.